### ISSUE TITLE
NMR shift wrappers are broken with Pandas 0.16

### ISSUE THREAD

**Issue by cing** [mdtraj/mdtraj#797]
Pretty minor issue pops up in Pandas 0.16 due to deprecating some vintage argument names.
```
  File "/Users/cing/anaconda/lib/python2.7/site-packages/mdtraj-1.4.0.dev0-py2.7-macosx-10.5-x86_64.egg/mdtraj/nmr/shift_wrappers.py", line 162, in chemical_shifts_shiftx2
    results = results.pivot_table(rows=["resSeq", "name"], cols="frame", values="SHIFT")
TypeError: pivot_table() got an unexpected keyword argument 'rows'
```
One line fix, rows to "index" and cols to "columns" fixes it. I'll submit a PR for due diligence, but now you can't claim to support "Pandas Version >= 0.9.0", the earliest you can do it 0.14.0!

**Comment by rmcgibbo** [comment]
grumble grumble...

**Comment by rmcgibbo** [comment]
(i feel like we've had a lot of pandas api breakage problems)

**Comment by kyleabeauchamp** [comment]
Yeah, this is my bad.

As an aside, for the Mol2 parser, we could possibly deprecate our
pandas code and use an optional parmed dependency.

**Comment by rmcgibbo** [comment]
That seems like overkill.

**Comment by rmcgibbo** [comment]
Fixed by #811.
